.gitignore

```diff
@@ -42,6 +42,9 @@ typings/
 # Optional npm cache directory
 .npm
 
+# Bothersome npm lockfile
+package-lock.json
```

Comment:
This should have been in the refactoring branch but I'll let it slide ;)

Reply:
Just merged the parent #2 so its changes should no longer be showing up here :3